server/nodejs.go

```diff
@@ -633,7 +633,7 @@ func toTypesPackageName(pkgName string) string {
 }
 
 func getDenoStdVersion() (version string, err error) {
-	resp, err := httpClient.Get("https://cdn.deno.land/std/meta/versions.json")
+	resp, err := http.Get("https://cdn.deno.land/std/meta/versions.json")
```

Comment:
Is this because of the HTTP_PROXY issue you mentioned? If so, want to try to add `Proxy` field to the Transport config, to see if it works for you? It's essentially dropping in this line:
```go
Proxy: http.ProxyFromEnvironment,
```
right below this line https://github.com/esm-dev/esm.sh/blob/46940b3b72de52cb2259d8973c9f959481e68dc4/server/query.go#L34

Reply:
@jimisaacs done, it seems to work. Thanks, go is not my usual language :)

Reply:
Mine neither to be honest, I approved and am running the workflow, but I think @ije should give this change a look. I don't think that setting should have any overhead given that it's the default of go, but I wouldn't know at the scale @ije is operating this service.

FYI, the check failure looks unrelated to your change.

Reply:
@ije added you here for this change (even though I realize i'll need to port it over to esmd regarding this PR https://github.com/esm-dev/esm.sh/pull/330)